src/components/seek_time/seek_time.js

```diff
@@ -89,8 +90,17 @@ export default class SeekTime extends UIObject {
       this.$el.css('left', "-100%")
     }
     else {
-      var seekTime = this.hoverPosition * this.duration
-      var currentSeekTime = formatTime(seekTime)
+      if (this.actualLiveTime) {
+        var d = new Date(), e = new Date(d);
+        var secondsSinceMidnight = (e - d.setHours(0,0,0,0)) / 1000;
+        var seekTime = (secondsSinceMidnight - this.duration) + (this.hoverPosition * this.duration)
+        if (seekTime < 0) {
+          seekTime = (3600 * 24) + seekTime
+        }
+      } else {
+        var seekTime = this.hoverPosition * this.duration
+      }
```

Comment:
What I meant was providing server time to what the stream is now at its freshest point, and then calculate what the beginning of the stream is by calculating how long since initiation and seek time. That would work I think.